### ISSUE TITLE
Cannot delete template

### ISSUE THREAD

**Issue by pofider** [jsreport/jsreport#161]
Copy from https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/jsreport/jMAx2WmTlow/tnfaZuUcDAAJ
```
2015-12-14T23:23:32.957Z - error: TypeError: Path must be a string. Received undefined
    at assertPath (path.js:9:11)
    at Object.posix.join (path.js:480:5)
    at Persistence.entryDirectory (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/lib/persistence.js:82:15)
    at /Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/lib/persistence.js:159:29
    at /Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/node_modules/async/lib/async.js:187:20
    at iterate (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/node_modules/async/lib/async.js:265:13)
    at Object.async.forEachOfSeries.async.eachOfSeries (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/node_modules/async/lib/async.js:284:9)
    at Object.async.forEachSeries.async.eachSeries (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/node_modules/async/lib/async.js:220:22)
    at Persistence.persistNewState (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/lib/persistence.js:155:9)
    at Datastore._remove (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/nedb/lib/datastore.js:605:20)
    at /Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/nedb/lib/executor.js:40:13
    at Immediate.q.process [as _onImmediate] (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/async/lib/async.js:731:21)
    at processImmediate [as _immediateCallback] (timers.js:384:17)
error!! TypeError: Path must be a string. Received undefined
    at assertPath (path.js:9:11)
    at Object.posix.join (path.js:480:5)
    at Persistence.entryDirectory (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/lib/persistence.js:82:15)
    at /Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/lib/persistence.js:159:29
    at /Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/node_modules/async/lib/async.js:187:20
    at iterate (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/node_modules/async/lib/async.js:265:13)
    at Object.async.forEachOfSeries.async.eachOfSeries (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/node_modules/async/lib/async.js:284:9)
    at Object.async.forEachSeries.async.eachSeries (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/node_modules/async/lib/async.js:220:22)
    at Persistence.persistNewState (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/jsreport-fs-store/lib/persistence.js:155:9)
    at Datastore._remove (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/nedb/lib/datastore.js:605:20)
    at /Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/nedb/lib/executor.js:40:13
    at Immediate.q.process [as _onImmediate] (/Users/JJH/Documents/Work/Conversant/reportingProject/jsreport/node_modules/async/lib/async.js:731:21)
```

**Comment by pofider** [comment]
Fixed in [jsreport-fs-store@0.2.1](https://www.npmjs.com/package/jsreport-fs-store)

This will be closed when we release jsreport with updated dependency.

**Comment by pofider** [comment]
Fix included in release 0.11.0
